src/IssueViz/Models/AnalysisIssueFlowVisualization.cs

```diff
@@ -23,7 +23,7 @@
 
 namespace SonarLint.VisualStudio.IssueVisualization.Models
 {
-    internal interface IAnalysisIssueFlowVisualization
+    public interface IAnalysisIssueFlowVisualization
```

Comment:
The error list/snapshot/analysis processing classes are all in the `Integration.VSIX` project, so they now need to be able to refer to the issue visualization types and converter. I've made the interfaces public but not the classes.